sentry_sdk/integrations/aiohttp.py

```diff
@@ -48,6 +53,52 @@
 TRANSACTION_STYLE_VALUES = ("handler_name", "method_and_path_pattern")
 
 
+def create_trace_config():
+    # type: () -> TraceConfig
+    async def on_request_start(session, trace_config_ctx, params):
+        # type: (ClientSession, SimpleNamespace, TraceRequestStartParams) -> None
+        hub = Hub.current
+        if hub.get_integration(AioHttpIntegration) is None:
+            return
+
+        method = params.method.upper()
+        request_url = str(params.url)
+
+        span = hub.start_span(
+            op=OP.HTTP_CLIENT, description="%s %s" % (method, request_url)
+        )
+        span.set_data("method", method)
+        span.set_data("url", request_url)
+
+        for key, value in hub.iter_trace_propagation_headers(span):
+            logger.debug(
+                "[Tracing] Adding `{key}` header {value} to outgoing request to {url}.".format(
+                    key=key, value=value, url=params.url
+                )
+            )
+            params.headers[key] = value
+
+        trace_config_ctx.span = span
+
+    async def on_request_end(session, trace_config_ctx, params):
+        # type: (ClientSession, SimpleNamespace, TraceRequestEndParams) -> None
+        if trace_config_ctx.span is None:
+            return
+
+        span = trace_config_ctx.span
+        span.set_data("status_code", int(params.response.status))
```

Comment:
We don't want to set this anymore since https://github.com/getsentry/sentry-python/pull/2113 -- please remove. The `set_http_status` below is sufficient